src/redux/reducers/extensions.test.js

```diff
@@ -0,0 +1,50 @@
+import { Reducer } from 'redux-testkit'
+
+import reducer, { defaultState } from './extensions'
+import * as actionTypes from './../constants/ActionTypes.js'
+
+describe('viewports reducer', () => {
+  it('should return the initial state', () => {
+    expect(reducer(undefined, {})).toEqual(defaultState)
```

Comment:
Ah. This is why you exported the default state.
This feels less brittle, and handy to have access to.